### PR TITLE
Adds ordeal text and fonts

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Adds flavor text to every ordeal and displays all of the information on every player's screen at the beginning of an ordeal. In addition, adds a folder for storing custom fonts. TG has its own system for fonts, but we cannot directly add to it so instead we can directly add genuine project moon fonts the old fashioned way.
Special thanks to discord user jtgszzs for helping me figure a lot of this out, would have taken me ages without their help.

https://github.com/vlggms/lobotomy-corp13/assets/35991533/62e24dc5-68e6-4a3c-bb90-18d62056ff7e

## Why It's Good For The Game
Huge boost to aesthetics, and makes players immediately aware of whatever threat is about to rip them limb from limb.

## Changelog
:cl:
add: Added floating ordeal blurbs
/:cl:

### CODE REVIEW

File: code/modules/ordeals/_ordeal.dm [hunk 1]
```diff
@@ -33,14 +35,16 @@
...
-	priority_announce(annonce_text, name, sound='sound/effects/meltdownAlert.ogg')
+	priority_announce(announce_text, name, sound='sound/effects/meltdownAlert.ogg')
 	/// If dawn started - clear suppression options
 	if(level == 1 && !istype(SSlobotomy_corp.core_suppression))
 		SSlobotomy_corp.ResetPotentialSuppressions()
-	if(annonce_sound)
-		for(var/mob/M in GLOB.player_list)
-			if(M.client)
-				M.playsound_local(get_turf(M), annonce_sound, 35, 0)
+	for(var/mob/M in GLOB.player_list)
```
Comment (Gboster-0) [inline code/modules/ordeals/_ordeal.dm:42]: can you please change this into NOT a single-letter variable?

Comment (Coxswain-Navigator) [reply code/modules/ordeals/_ordeal.dm:42]: This isn't really a big deal but I certainly can.

File: icons/fonts/font.dm [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// Simple datum to have fonts loaded in
+// Hopefully one day I can make this a big cleaner looking - Coxswain
```
Comment (LanceSmites328) [inline icons/fonts/font.dm:2]:
```suggestion
// Hopefully one day I can make this a bit cleaner looking - Coxswain
```

**Review by LanceSmites328 — APPROVED** [review]
Fuck yeah

**Review by EgorDinamit — APPROVED** [review]
If this PR is ready - it's good to go.

**Pull request merged**